pkgs/desktops/lxqt/default.nix

```diff
@@ -71,7 +71,7 @@ let
       pkgs.lxmenu-data
       pkgs.menu-cache
       pkgs.openbox # default window manager
-      qt5.qtsvg # provides QT5 plugins for svg icons
+      pkgs.qt5.qtsvg # provides QT5 plugins for svg icons
```

Comment:
It does not work if `qtsvg` is used instead of `pkgs.qt5.qtsvg`:

```
$ nixos-rebuild build-vm -I nixpkgs=/alt/nixpkgs -I nixos-config=/alt/nixos/configuration.lxqt.nix
building Nix...
building the system configuration...
error: undefined variable ‘qtsvg’ at /alt/nixpkgs/pkgs/desktops/lxqt/default.nix:74:7
```